source/agora/node/Ledger.d

```diff
@@ -760,7 +760,9 @@ public class Ledger
         if (auto fail_reason = this.getValidTXSet(data, tx_set))
             return fail_reason;
 
-        size_t active_enrollments = enroll_man.getValidatorCount(expect_height);
+        long active_enrollments = enroll_man.getValidatorCount(expect_height) -
+            data.missing_validators.length;
+        assert(active_enrollments >= 0);
```

Comment:
```suggestion
        size_t active_enrollments = enroll_man.getValidatorCount(expect_height);
        assert(active_enrollments >= data.missing_validators.length;
```

It's usually better to compare two values instead of subtract and compare two zero, since you don't risk overflowing when comparing.

Reply:
Updated.